packages/components/src/checkbox-control/stories/index.tsx

```diff
@@ -0,0 +1,126 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import CheckboxControl from '..';
+
+const meta: ComponentMeta< typeof CheckboxControl > = {
+	component: CheckboxControl,
+	title: 'Components/CheckboxControl',
+	argTypes: {
+		onChange: {
+			action: 'onChange',
+		},
+		checked: {
+			control: { type: null },
+		},
+		help: { control: { type: 'text' } },
+	},
+	parameters: {
+		controls: {
+			expanded: true,
```

Comment:
```suggestion
			expanded: true,
			exclude: [ 'heading' ],
```

This should remove `heading` from the controls and docs tables further discouraging its use.